config/boot.rb

```diff
@@ -1,13 +1,3 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
-
```

Comment:
Is there any reason not to leave this in and just update it to work with Rails 5? My thought is, this way you don't have to always remember to start your server with `-b 0.0.0.0 -p 8080` and wonder why it's not working if you forget to do that. Thoughts?

```
require 'rails/command
require 'rails/commands/server/server_command'
module Rails		
  class Server		
    alias :_default_options :default_options		
    def default_options		
      _default_options.merge!(Host:'0.0.0.0', Port:8080)		
    end		
  end		
end
```

Reply:
This is no longer supported via Rails 5.1. The internal structure changed, and I did not see a way to reimplement this. I would also suggest this method of monkey patching is less than ideal and prone to breaking in such cases like an update or patch of rails.